src/components/CardList.js

```diff
@@ -1,21 +1,20 @@
 import '../styles/CardList.css';
 import Card from './Card';
 
-const CardList = ({ folderInfo }) => {
-  const card = folderInfo?.folder.links;
-  console.log(card);
+const CardList = ({ folderLink, folderInfo, isIconVisible }) => {
+  const card = folderLink || folderInfo?.folder.links;

```

Comment:
P1:
이름 신경써주세요~ card에 card 속성은 어떤 값인지 알 수가 없습니다.
또한 folderLink 혹은 folderInfo 객체의 folder links보다 외부에서 해당하는 링크를 넘겨주는 방식이면 좋겠네요.

```suggestion
/** <CardList folderLinks={folderInfo.folder.links} /> */
const CardList = ({ folderLinks, isIconVisible }) => {
```